src/mongocrypt-crypto.c

```diff
@@ -626,29 +651,28 @@ _encrypt_step (_mongocrypt_crypto_t *crypto,
  */
 static bool
 _hmac_step (_mongocrypt_crypto_t *crypto,
-            const _mongocrypt_buffer_t *mac_key,
-            const _mongocrypt_buffer_t *associated_data,
-            const _mongocrypt_buffer_t *ciphertext,
+            _mongocrypt_mac_format_t mac_format,
+            _mongocrypt_hmac_type_t hmac,
+            const _mongocrypt_buffer_t *Km,
```

Comment:
You need to update the comment if you keep the new variables names.

Reply:
Good catch